login.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+DVL_PATH="${1:-/home/witek/Projects/devilbox}"
```

Comment:
We've already discussed in this PR: https://github.com/cytopia/devilbox/pull/447 that those scripts should not be part. Especially this on already exists as `shell.sh`: https://github.com/cytopia/devilbox/blob/master/shell.sh

Please remove `login.sh`